pyteal/ast/app_test.py

```diff
@@ -92,38 +134,60 @@ def test_local_get_ex():
         TealOp(None, Op.store, expr.slotOk),
         TealOp(None, Op.store, expr.slotValue)
     ])
-    
+
+    actual, _ = expr.__teal__(options)
+    actual.addIncoming()
+    actual = TealBlock.NormalizeBlocks(actual)
+
+    with TealComponent.Context.ignoreExprEquality():
+        assert actual == expected
+
+def test_local_get_ex_direct_ref():
+    args = [Txn.sender(), Int(6), Bytes("key")]
+    expr = App.localGetEx(args[0], args[1], args[2])
+    assert expr.type_of() == TealType.none
+    assert expr.value().type_of() == TealType.anytype
+
+    expected = TealSimpleBlock([
+        TealOp(args[0], Op.txn, "Sender"),
+        TealOp(args[1], Op.int, 6),
+        TealOp(args[2], Op.byte, "\"key\""),
+        TealOp(expr, Op.app_local_get_ex),
+        TealOp(None, Op.store, expr.slotOk),
+        TealOp(None, Op.store, expr.slotValue)
+    ])
+
     actual, _ = expr.__teal__(options)
     actual.addIncoming()
     actual = TealBlock.NormalizeBlocks(actual)
-    
+
     with TealComponent.Context.ignoreExprEquality():
         assert actual == expected
 
 def test_local_get_ex_invalid():
     with pytest.raises(TealTypeError):
-        App.localGetEx(Txn.sender(), Int(0), Bytes("key"))
-    
+        App.localGetEx(Txn.sender(), Int(0), Int(0x123456))
+
     with pytest.raises(TealTypeError):
         App.localGetEx(Int(0), Bytes("app"), Bytes("key"))
 
     with pytest.raises(TealTypeError):
-        App.localGetEx(Int(0), Int(0), Int(1))
+        App.localGetEx(Txn.sender(), Int(0), Int(1))
```

Comment:
It seems like this case is testing the same as `App.localGetEx(Txn.sender(), Int(0), Int(0x123456))`? If so, it can be removed.

Reply:
Yea it is a duplicated case. Removed now.